### ISSUE TITLE
Movement/prediction stutters

### ISSUE THREAD

**Issue by mikota** [jdolan/quetoo#682]
There are visual stutters while moving. To me it looks like prediction errors but I'm no expert.

The issue is shown in the video below:

https://github.com/jdolan/quetoo/assets/13764327/be7824d8-0d2b-4a2e-a055-159127feda19

This is an edited video, slowed down 2.75x to 60fps from the original recording in 165 fps. In the first part of the video I simply move my mouse arround, demonstrating that this isn't some general framepacing/rendering issue. In the second part of the video I'm moving around, and you can clearly see the stuttering.

The original non-cut, full speed at 165fps video can be downloaded [here](https://mega.nz/file/zown1T4S#R-1zljgOE96NGll3IcDnni81YtBDLXS8HN0qLgw_9gY). I repeat the same experiment multiple times in the same video.

The issue occurs in all video modes, vsync on/off/immediate. The issue becomes more noticable the faster I'm moving.

System info:
Computer Information:
Manufacturer: Gigabyte Technology Co., Ltd.
Model: B660M DS3H AX DDR4
Form Factor: Desktop
No Touch Input Detected
Processor Information:
CPU Vendor: GenuineIntel
CPU Brand: 12th Gen Intel(R) Core(TM) i5-12400F
CPU Family: 0x6
CPU Model: 0x97
CPU Stepping: 0x5
CPU Type: 0x0
Speed: 2496 MHz
12 logical processors
6 physical processors
Hyper-threading: Supported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Unsupported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Supported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Supported
AVX512VNNI: Supported
SHA: Supported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
Operating System Version:
Windows 10 (64 bit)
NTFS: Supported
Crypto Provider Codes: Supported 311 0x0 0x0 0x0
Video Card:
Driver: NVIDIA GeForce GTX 1050 Ti
DirectX Driver Name: nvldumd.dll
Driver Version: 31.0.15.3623
DirectX Driver Version: 31.0.15.3623
Driver Date: 6 8 2023
OpenGL Version: 4.6
Desktop Color Depth: 32 bits per pixel
Monitor Refresh Rate: 60 Hz
DirectX Card: NVIDIA GeForce GTX 1050 Ti
VendorID: 0x10de
DeviceID: 0x1c82
Revision: 0xa1
Number of Monitors: 2
Number of Logical Video Cards: 2
No SLI or Crossfire Detected
Primary Display Resolution: 1920 x 1080
Desktop Resolution: 3840 x 1080
Primary Display Size: 20.51" x 11.54" (23.50" diag), 52.1cm x 29.3cm (59.7cm diag)
Primary Bus: PCI Express 8x
Primary VRAM: 4095 MB
Supported MSAA Modes: 2x 4x 8x
Sound card:
Audio device: Headphones (High Definition Aud
Memory:
RAM: 16225 Mb
VR Hardware:
VR Headset: None detected
Miscellaneous:
UI Language: English
Media Type: Undetermined
Total Hard Disk Space Available: 3323339 MB
Largest Free Hard Disk Block: 1412293 MB
OS Install Date: Oct 25 2023
Game Controller: None detected
MAC Address hash: 9d4a496d14356ebee8d2fe4a3b34092cfe1fd5a4
Storage:
Disk serial number hash: 558f35cf
Number of SSDs: 2
SSD sizes: 480G,240G
Number of HDDs: 2
HDD sizes: 2000G,1000G
Number of removable drives: 0

**Comment by jdolan** [comment]
With `cl_predict 0` does this go away?

**Comment by mikota** [comment]
> With `cl_predict 0` does this go away?

yes. note that I'm running on the build from the site which is pretty old

**Comment by jdolan** [comment]
@Paril Do you see this on Windows? I have never seen this on macOS or Linux. My hardware is pretty comparable to @mikota 's, but.. I can't reproduce this. At 40fps, 60fps, 125fps and uncapped. 🤷‍♂️
